### ISSUE TITLE
codec support

### ISSUE THREAD

**Issue by morekid** [processing/processing-video#111]
I'm having difficulties finding the right codec settings for use with this library. I tried a number of options with H.264 on Apple Compressor but I get no playback in Processing.

Is there a guide to how to properly export media for this?

I have also tried exporting HEVC but it's not clear whether the gstreamer plugin for this is available within this library.

**Comment by codeanticode** [comment]
@morekid have you tried the latest beta of v2?

**Comment by morekid** [comment]
Thanks for suggesting this, will try the beta.

**Comment by codeanticode** [comment]
Try [beta3](https://github.com/processing/processing-video/releases/tag/r5-v2.0-beta3) instead, it fixes several issues and bundles the latest version of gstreamer (1.16) for mac and windows.

**Comment by codeanticode** [comment]
@morekid are you able to play your videos using the latest beta of the library (btw, beta4 is available now, next should be 2.0 stable). Both beta3 and 4 use GStreamer 1.1.6.0, which is the most recent GStreamer version.

**Comment by morekid** [comment]
Yes I could. All H.264 options I tried work perfectly with beta4 + gstreamer 1.1.6.0.

I only had issues with lossless H.264 `ffmpeg -crf 22`, see below. However that's something that also QT struggles with (VLC plays it fine).

![image](https://user-images.githubusercontent.com/632296/65754938-6617af80-e112-11e9-99f9-8a06e161b6ec.png)
